tests/models/zoedepth/test_modeling_zoedepth.py

```diff
@@ -300,8 +300,8 @@ def check_target_size(
             out_l_reduced = torch.nn.functional.interpolate(
                 out_l.unsqueeze(0).unsqueeze(1), size=img.size[::-1], mode="bicubic", align_corners=False
             )
-            self.assertTrue((np.array(out_l.shape)[::-1] == np.array(img.size) * 2).all())
```

Comment:
looks we remove one check here? As long as you believe this check is not necessary, good for me.

Reply:
this check compares shapes, I believe `torch.testing.assert_close` takes this into account as well